src/components/FitClassifierDialog/TrainingHistoryPlot/TrainingHistoryPlot.tsx

```diff
@@ -0,0 +1,124 @@
+import { Container, Typography } from "@mui/material";
+import { ResponsiveLine, Serie } from "@nivo/line";
+
+type TrainingHistoryPlotProbs = {
+  metric: string;
+  trainingValues: { x: number; y: number }[];
+  validationValues: { x: number; y: number }[];
+  dynamicYRange?: boolean;
+};
+
+export const TrainingHistoryPlot = (props: TrainingHistoryPlotProbs) => {
+  const {
+    metric,
+    trainingValues,
+    validationValues,
+    dynamicYRange = false,
+  } = props;
+
+  const trainingData: Serie = {
```

Comment:
Hmm, I'm wondering if it's "safe" to use the `Serie` type explicitly. They clearly [export it](https://github.com/plouc/nivo/blob/2763c4a28e5376457823b3be7940745c7f6feb8c/packages/line/index.d.ts#L45), but they make no mention of it in the [API docs](https://nivo.rocks/line/api/). I'm probably being paranoid, but I'm nervous that it may be for internal use only (and therefore subject to change in future versions).

Reply:
Yes the types are not documented very well. Should I just use `any`?